src/main/java/frc/robot/subsystems/superstructure/Hood.java

```diff
@@ -5,47 +5,49 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-package frc.robot.subsystems;
+package frc.robot.subsystems.superstructure;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANEncoder;
 import com.revrobotics.CANSparkMaxLowLevel;
-public class Hood extends SubsystemBase {
-
-  
-  CANSparkMaxLowLevel.MotorType brushless = CANSparkMaxLowLevel.MotorType.kBrushless;
-  private CANSparkMax hoodMain;
-  private CANEncoder hoodEncoder;
-
-  public Hood() {
-
-      // instantiate the neo550 and encoder
-      hoodMain = new CANSparkMax(Constants.Hood.HOOD_MOTOR_ID, brushless);
-      hoodEncoder = new CANEncoder(hoodMain);
-  }
-
-  // raises the hood, lowers angle of shot
-  public void raiseHood() {
-      hoodMain.set(Constants.Hood.HOOD_MAX_SPEED);
-  }
 
-  // lowers hood, increases angle of shot
-  public void lowerHood() {
-      hoodMain.set(-Constants.Hood.HOOD_MAX_SPEED);
-  }
-
-
-  protected void useOutput() {}
-
-  protected double getMeasurement() { 
-      return 0.0;
-      //CHANGE
-  }
+public class Hood extends SubsystemBase {
 
-  @Override
-  public void periodic() {
-    // This method will be called once per scheduler run
-  }
+    // fields
+    private CANSparkMax hoodMain; // main motor
+    private CANEncoder hoodEncoder; // main motor encoder
+
+    public Hood() {
+        // instantiate neo550 and encoder
+        hoodMain = new CANSparkMax(Constants.Hood.HOOD_MOTOR, Constants.Hood.MOTOR_TYPE);
+        hoodEncoder = hoodMain.getEncoder();
+    }
+
+    // raises the hood, lowers angle of shot
+    public void raiseHood() {
+        hoodMain.set(Constants.Hood.HOOD_MAX_SPEED);
+    }
+
+    // lowers hood, increases angle of shot
+    public void lowerHood() {
+        hoodMain.set(-Constants.Hood.HOOD_MAX_SPEED);
+    }
+
+    // stop hood movement
+    public void stopHood() { hoodMain.set(0); }
+
+    protected void useOutput() {
+    }
```

Comment:
is this method called? if not, remove this since it's empty

Reply:
maybe @override ?